sdk/typescript/src/types/events.ts

```diff
@@ -89,6 +89,12 @@ export type SuiEventEnvelope = {
     event: SuiEvent
 }
 
+export type SuiEvents = SuiEventEnvelope[];
+
 export type SubscriptionId = number;
 
-export type SubscriptionEvent = { subscription: SubscriptionId, result: SuiEventEnvelope };
\ No newline at end of file
+export type SubscriptionEvent = { subscription: SubscriptionId, result: SuiEventEnvelope };
+
+export const DEFAULT_RESULT_COUNT = 1000;
```

Comment:
Can we mirror the value in https://github.com/MystenLabs/sui/blob/e12f8c58ef7ba17205c4caf5ad2c350cbb01656c/crates/sui-json-rpc/src/api.rs#L27 ?
```suggestion
// mirrors the value defined in https://github.com/MystenLabs/sui/blob/e12f8c58ef7ba17205c4caf5ad2c350cbb01656c/crates/sui-json-rpc/src/api.rs#L27
export const EVENT_QUERY_MAX_LIMIT = 100;
```